bundles/org.openhab.binding.meater/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -91,7 +91,7 @@
 	<channel-type id="cookRemainingTime">
 		<item-type>Number:Time</item-type>
 		<label>Cook Remaining Time</label>
-		<description>Remaining time in seconds. When unknown/calculating default is used. Default: UnDefType.UNDEF</description>
+		<description>Remaining time in seconds or UNDEF when unknown.</description>
```

Comment:
i18n properties should be regenerated after this change.

Reply:
Fixed!